src/controller/python/chip/clusters/Attribute.py

```diff
@@ -29,6 +29,7 @@
 import inspect
 import sys
 import logging
+import ipdb
```

Comment:
is this import needed?

Reply:
Seems this is only used for debugging, I think this is added by mistake